src/protocols/secure_channel/CASESession.cpp

```diff
@@ -160,14 +172,15 @@ CHIP_ERROR CASESession::ToSerializable(CASESessionSerializable & serializable)
     VerifyOrReturnError(CanCastTo<uint64_t>(peerNodeId), CHIP_ERROR_INTERNAL);
 
     memset(&serializable, 0, sizeof(serializable));
```

Comment:
We need a version byte to `serializable` so that we don't get really weird crashes if previous versions of data exist. If the version mismatches, a record should be expunged.